matter/wifi/ethernetif.cpp

```diff
@@ -92,45 +97,51 @@ static void low_level_init(struct netif *netif)
   /* Accept broadcast address and ARP traffic */
   netif->flags |= NETIF_FLAG_BROADCAST | NETIF_FLAG_ETHARP | NETIF_FLAG_IGMP;
 
-  /* Set netif link flag */
-  //    netif->flags |= NETIF_FLAG_LINK_UP|NETIF_FLAG_UP;
 }
 
-/*
+/********************************************************************************
+ * @fn  static void low_level_input(struct netif *netif, uint8_t *b, uint16_t len)
+ * @brief  
  * Make PBUF out of a linear buffer - that can be fed into lwip
- */
+ * @param[in]  netif: the already initialized lwip network interface structure
+ * @param[in]  len: length
+ * @return
+ *     None
+ ************************************************************************************/
 static void low_level_input(struct netif *netif, uint8_t *b, uint16_t len)
 {
   struct pbuf *p, *q;
   uint32_t bufferoffset;
 
   if (len <= 0)
```

Comment:
It's a good practice to always use brackets {} in conditional statements.

Reply:
done